models/insertComment.js

```diff
@@ -0,0 +1,14 @@
+const db = require('../db/connection')
+
+function insertComment({body, votes, author, article_id, created_at}) {
+    return db.query(`
+        INSERT INTO comments (body, votes, author, article_id, created_at)
+        VALUES ($1, $2, $3, $4, $5)
```

Comment:
Votes and created_at has default value when inserting into the table, so no need to include them